aries-site/src/pages/templates/side-to-side-navigation.mdx

```diff
@@ -90,12 +86,13 @@ export const data = {
 
 export const TabCaption = () => (
   <NameValueList
-    layout="grid"
```

Comment:
layout = "grid" is necessary to control the responsive behavior of the NameValueList.  Proposing we leave this in, but just wrap the entire NameValueList in a Box.

```
<Box>
  <NameValueList layout="grid' ...>
  ...
  </NameValueList>
</Box>